### PR TITLE
refactor(GlobalHeader): Clean up experiment in favor of variant

### PR DESCRIPTION
**Pull request by zippyzow**
### Overview
V3 experiment is complete and we are confident in moving forward with the variant, let's remove the useNewCatalogDropdown and default to the new Catalog and Resources dropdowns.

### PR Checklist

- [x] Related to designs: DISC-786
- [x] Related to JIRA ticket: [DISC-768]
- [x] I have run this code to verify it works
- [x] This PR includes unit tests for the code change
- [x] This PR includes testing instructions tests for the code change
- [x] The alpha package of this PR is passing end-to-end tests in all relevant Codecademy repositories

#### Testing Instructions
Check out the PR envs and make sure you see the new dropdowns for Catalog and Resources.
Screen Shot 2022-09-27 at 4 05 38 PM

[Don't make me tap the sign.](https://i.imgur.com/sy93D9I.png)

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| Monolith     | [Monolith PR](https://github.com/codecademy-engineering/Codecademy/pull/32742)  | [Monolith Env](https://pr-32742-monolith.dev-eks.codecademy.com/) |
| Portal       | [Portal Link](https://github.com/codecademy-engineering/mono/pull/247)  | [Portal Env]( https://tengu.codecademy.com/catalog?PR_ENV=portal-app-pr-247)   |

[DISC-768]: https://codecademy.atlassian.net/browse/DISC-768?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.207-alpha.cda16b.0<br/>@codecademy/gamut-labs@33.5.3-alpha.cda16b.0<br/>@codecademy/styleguide@61.5.14-alpha.cda16b.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://633216dc8ed2840e788fa5cb--gamut-preview.netlify.app>https://633216dc8ed2840e788fa5cb--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/633216dc8ed2840e788fa5cb>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.207-alpha.4145b0.0<br/>@codecademy/gamut-labs@33.5.3-alpha.4145b0.0<br/>@codecademy/styleguide@61.5.14-alpha.4145b0.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://633317acb5ec63588dfd7f74--gamut-preview.netlify.app>https://633317acb5ec63588dfd7f74--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/633317acb5ec63588dfd7f74>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.207-alpha.e7f9eb.0<br/>@codecademy/gamut-labs@33.5.3-alpha.e7f9eb.0<br/>@codecademy/styleguide@61.5.14-alpha.e7f9eb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://633333f09bb5790ad8219000--gamut-preview.netlify.app>https://633333f09bb5790ad8219000--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/633333f09bb5790ad8219000>Deploy Logs</a></p>

File: packages/gamut-labs/src/GlobalHeader/__tests__/GlobalHeader.test.tsx [hunk 1]
```diff
@@ -430,7 +372,7 @@ describe('GlobalHeader', () => {
       });
 
       it('renders business solutions', () => {
-        const { view } = renderView(freeHeaderProps);
+        const { view } = renderView(proHeaderProps);
```
Comment (borisonr) [inline packages/gamut-labs/src/GlobalHeader/__tests__/GlobalHeader.test.tsx:375]: how come we needed this change?

**Review by borisonr — APPROVED** [review]
lgtm!

File: packages/gamut-labs/src/GlobalHeader/__tests__/GlobalHeader.test.tsx (hunk 1 above)
Comment (zippyzow) [reply packages/gamut-labs/src/GlobalHeader/__tests__/GlobalHeader.test.tsx:375]: Just confirmed with Jahaziel that `proHeaderProps` was intended so good to keep this change

**Review by dreamwasp — APPROVED** [review]
🙌

**Pull request merged**